pytest_robotframework/_internal/plugin.py

```diff
@@ -592,3 +597,44 @@ def pytest_runtest_protocol(item: Item):
         )
         return True
     return None
+
+
+def _escape(text: str, escapes: Sequence[tuple[str, str]] = markuputils._generic_escapes) -> str:  # pyright: ignore[reportPrivateUsage]
+    esc = "\N{ESCAPE}"
+    if esc in text:
+        if _markup_ansi:
+            text = Ansi2HTMLConverter(inline=True, escaped=False).convert(text, full=False)
+        else:
+            text = re.sub(rf"{esc}\[\d\d?(;\d\d(;\d\d)?)?m\[?", "", text)
+    return markuputils_escape(text, escapes)
+
+
+markuputils_escape = markuputils._escape  # pyright: ignore[reportPrivateUsage, reportUnknownVariableType, reportUnknownMemberType]
+markuputils._escape = _escape  # pyright: ignore[reportPrivateUsage]
+
+_markup_ansi = True
+
+
+@patch_method(_MarkupWriter)
+def element(
```

Comment:
maybe we should use a pre-rebot modifier instead, but this can be done in a follow up PR

Reply:
#206

Reply:
erm actually a listener